addons/binding/org.openhab.binding.vitotronic/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,489 @@
+        <label>Party</label>
+        <description>Managed Temperatur</description>
+        <category>Temperature</category>
+        <state pattern="%.1f °C" readOnly="false"/>
+    </channel-type>
+    
+     <channel-type id="save_temp">
+        <item-type>Number</item-type>
+        <label>Save</label>
+        <description>Managed Temperatur</description>
+        <category>Temperature</category>
+        <state pattern="%.1f °C" readOnly="false"/>
+    </channel-type>
+    
+    <channel-type id="room_temp">
+        <item-type>Number</item-type>
+        <label>Room</label>
+        <description>Managed Temperatur</description>
+        <category>Temperature</category>
+        <state pattern="%.1f °C" readOnly="false"/>
+    </channel-type>
+    
+    <channel-type id="gradient">
+        <item-type>Number</item-type>
+        <label>Gradient</label>
+        <description>The gradient relativ to outseite temperature.</description>
+        <state pattern="%.1f" readOnly="true"/>
+    </channel-type>
+    
+    <channel-type id="niveau">
+        <item-type>Number</item-type>
+        <label>Niveau</label>
+        <description>The niveau relativ to outseite temperature.</description>
+        <state pattern="%.1f" readOnly="true"/>
+    </channel-type>
+ 
+    
+    <!-- Channel Type for solar -->
+
+     <channel-type id="collector_temp">
+        <item-type>Number</item-type>
+        <label>Collector</label>
+        <description>Temperatur sensor</description>
+        <category>Temperature</category>
+        <state pattern="%.1f °C" readOnly="true"/>
+    </channel-type>
+ 
+    <channel-type id="storagetank_temp">
+        <item-type>Number</item-type>
+        <label>Storagetank</label>
+        <description>Temperatur sensor</description>
+        <category>Temperature</category>
+        <state pattern="%.1f °C" readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="bufferload">
+        <item-type>Switch</item-type>
+        <label>Buffer load pump</label>
+        <description> state</description>
+        <state readOnly="true"/>
+    </channel-type>
+    
+    <channel-type id="loadsuppression">
+        <item-type>Switch</item-type>
+        <label>Load suppression</label>
+        <description> state</description>
+        <state readOnly="true"/>
+    </channel-type>
+    
+     
+    <channel-type id="producedheat">
+        <item-type>Number</item-type>
+        <label>Produced heat</label>
+        <description>Produced heat since starting solar system</description>
+        <state pattern="%.2f KWh" readOnly="true"/>
+    </channel-type>
+    
+    <!-- single types -->
+    
+    
+    <channel-type id="temperature">
+        <item-type>Number</item-type>
+        <label>Temperature</label>
+        <description>Temperatur sensor</description>
+        <category>Temperature</category>
+        <state pattern="%.1f °C" readOnly="true"/>
+    </channel-type>
+    
+    <channel-type id="value">

```

Comment:
couldn't it then be named "valve" instead (which has a "value" as a state)? Because "value" almost fits for any channel, so "valve" would be a bit more specific. And If the description would be slightly more than one word, it could help the untrained user like me as well ;-)